Cargo.toml

```
@@ -24,8 +24,8 @@ categories = [
 [dependencies]
 # 0.19 is when file handle started to return Arc instead of Box
 # https://github.com/quickwit-oss/tantivy/commit/775e936f7d8e461b7267bb13763db37c1c45afb8
-tantivy = ">=0.19"
-object_store = ">=0.6.1"
+tantivy = "^0.19"
+object_store = "^0.6.1"
```

Nit: canonically, you don't need to tilde's. But feel free to keep them for clarity if you want.
```suggestion
tantivy = "0.19"
object_store = "0.6.1"
```